src/file_pipe_log.rs

```diff
@@ -25,6 +25,8 @@ use crate::pipe_log::{FileId, LogQueue, PipeLog, SequentialReplayMachine};
 use crate::reader::LogItemBatchFileReader;
 use crate::util::InstantExt;
 use crate::{Error, Result};
+use std::fs::File;
```

Comment:
It should be put above with other `std::xx`

Reply:
Sure, Seems that run `cargo clippy` and `cargo build` can't detect this problem

Reply:
Yes, clippy only sorts the imports, not group them.